pkg/ruler/manager_test.go

```diff
@@ -40,21 +45,36 @@ func TestSyncRuleGroups(t *testing.T) {
 				User:      user,
 			},
 		},
+		user1: {
+			&rulespb.RuleGroupDesc{
+				Name:      "group1",
+				Namespace: "ns",
+				Interval:  1 * time.Minute,
+				User:      user1,
+			},
+		},
 	}
 	m.SyncRuleGroups(context.Background(), userRules)
 
 	mgr := getManager(m, user)
```

Comment:
[nit] Same here. Let's use "1" for "user1" and "2" for "user2".

```suggestion
	mgr1 := getManager(m, user)
```